### ISSUE TITLE
Missing hapService.BatteryService

### ISSUE THREAD

**Issue by lboue** [ilcato/homebridge-Fibaro-HC2#137]
Hello,

Could you tell me how to add the _hapService.BatteryService_ service to the accessory in the [createShadowAccessory ](https://github.com/ilcato/homebridge-Fibaro-HC2/blob/master/dist/shadows.js#L155)method?
The goal is to expose Battery Level 

- Battery Service is described on page 230, [HAP-Specification-Non-Commercial-Version](https://community.ubnt.com/ubnt/attachments/ubnt/airVision_Ideas/2505/1/HAP-Specification-Non-Commercial-Version.pdf)
- Battery Level is described on page 167, [HAP-Specification-Non-Commercial-Version](https://community.ubnt.com/ubnt/attachments/ubnt/airVision_Ideas/2505/1/HAP-Specification-Non-Commercial-Version.pdf)

** homebridge-mi-aqara**
I found an implementation example here with the homebridge-mi-aqara plugin and the [SmokeDetectorParser.js](https://github.com/YinHangCode/homebridge-mi-aqara/blob/9ca6907b6d26248126aae816af8ad6f33446cf1a/parser/SmokeDetectorParser.js#L43) class 

**homebridge-hue example**
https://github.com/ebaauw/homebridge-hue/blob/314719ea2771381289b65c828471646e085a5663/lib/HueAccessory.js#L292
```
HueAccessory.prototype.getBatteryService = function (battery) {
  if (this.batteryService == null) {
    this.batteryService = new Service.BatteryService(this.name)
    this.batteryService.getCharacteristic(Characteristic.ChargingState)
      .setValue(Characteristic.ChargingState.NOT_CHARGEABLE)
    this.state.battery = battery
    this.checkBattery(battery)
  }
  return this.batteryService
}
```

Can you confirm it should be declared like this?
`new ShadowService(new hapService.BatteryService(device.name), [hapCharacteristic.StatusLowBattery, hapCharacteristic.BatteryLevel, hapCharacteristic.ChargingState])
`
Regards

**Comment by ilcato** [comment]
This is correct. You must also implement the corresponding functions in getFunctions.ts.

**Comment by hovorkap** [comment]
I have a working solution for this, expect a PR shortly
https://github.com/hovorkap/homebridge-Fibaro-HC2/pull/2

**Comment by lboue** [comment]
Fixed by PR #150.
